app/controllers/home.js

```diff
@@ -10,19 +10,35 @@ export default Ember.Controller.extend({
     showNotSavedModal: false,
     published: false,
     saved: false,
+    unpublishedChanges:  Ember.computed('editMode', function() {
+        return this.store.findRecord('home', this.get('model.guid')).then((record)=>{ 
+            
+            let unpublishedPageData = record.get('unpublishedPageData');
+            let pageData =  record.get('pageData');
+
+            if(_.isEqual(unpublishedPageData, pageData)){
+                this.set('unpublishedChanges', false)
+            }else{
+                this.set('unpublishedChanges', true)
+
+            }
+        });
+
+        //return isPublished
+    }),
     savedPageData: "",
     isAdmin: Ember.computed('node', function(){
         this.send('canUserEdit')
         return this.get('model.node.currentUserPermissions').includes('admin');
     }),
-    firebaseData: Ember.observer('editMode', function(){
+    databaseData: Ember.observer('editMode', function(){
         $(document).ready(()=>{
             this.send('canUserEdit')
 
-            let firebaseDB = this.store.findRecord('home', this.get('model.guid'))
+            let database = this.store.findRecord('home', this.get('model.guid'))
```

Comment:
This is not a good name, can you rename to what it brings back?  like pagesData?

Reply:
it is the database node though not the page data, maybe backendNode